crates/router/src/connector/klarna.rs

```diff
@@ -306,6 +306,7 @@ impl
                     | storage_enums::PaymentExperience::OneClick
                     | storage_enums::PaymentExperience::RedirectToUrl,
                     api_models::enums::PaymentMethodType::Ach
+                    | api_models::enums::PaymentMethodType::AchBankTransfer
```

Comment:
We already have `Ach`, `Bacs` and `Sepa` in the `PaymentMethodType` enum. We do not need separate variants specifically for bank transfers. Wherever the context of the payment method is needed alongside Ach, Bacs and Sepa, please pass an instance of the `PaymentMethod` enum (here `PaymentMethod::BankTransfer`)

Reply:
https://github.com/juspay/hyperswitch/pull/3113#discussion_r1429191182